.github/workflows/continuous-integration.yml

```diff
@@ -1136,7 +1136,7 @@ jobs:
   deploy:
     name: Deploy
     runs-on: ubuntu-latest
-    if: ${{ always() && github.ref == 'refs/heads/master' && needs.get-latest-run-number.result == 'success' && needs.get-latest-run-number.outputs.latest_run_number == github.run_number}}
+    if: ${{ always() && github.ref == 'refs/heads/master' && needs.get-latest-run-number.result == 'success' }}
```

Comment:
This should work as long as the outdated GH API result for latest master workflow run, was a successful run

Reply:
> This should work as long as the outdated GH API result for latest master CI workflow run, was a successful run
oh, I should have also removed the `== 'success'` check?

Reply:
Maybe but I don't think it will matter since the latest run returned (from 6 days ago) is successful: https://github.com/department-of-veterans-affairs/vets-website/actions/workflows/continuous-integration.yml?query=branch%3Amaster